docs/spec/spec.md

```diff
@@ -1,10 +1,35 @@
-## Overview
+# Specification: Ballerina Yaml Library
 
-Ballerina YAML parser provides APIs to convert a YAML configuration file to `json`, and vice-versa. The module supports both the functions of read and write either a single YAML document or a YAML stream.
+_Owners_: @shafreenAnfar @MadhukaHarith92  
+_Reviewers_: @shafreenAnfar  
+_Created_: 2023/04/04  
+_Updated_: 2023/04/04  
+_Edition_: Swan Lake
+
+## Introduction
+This is the specification for the Yaml standard library of [Ballerina language](https://ballerina.io/), which provides APIs to convert a YAML configuration file to json, and vice-versa. The module supports both the functions of read and write either a single YAML document or a YAML stream.
+
```

Comment:
Can't we directly bind it to a record?

Reply:
Currently, the APIs are bound to JSON. Will create and issue to bind it directly to a record.

Reply:
Currently, APIs are bound to JSON. Will create an issue to directly bind it to a record.